### PR TITLE
update intro mapedit post for camelCase

### PR DESCRIPTION
**Pull request by timelyportfolio**
Please wait for review by @tim-salabim and pull request merge.  Thanks!

Update the [Introduction to mapedit post](http://r-spatial.org/r/2017/01/30/mapedit_intro.html) for new camelCase `editMap` and `selectMap` https://github.com/r-spatial/mapedit/pull/22 and https://github.com/r-spatial/mapedit/issues/20.  Also record new screenshot gifs and png affected by the change.

### CODE REVIEW

**Comment by tim-salabim** [comment]
I haven't updated on my machine but from looking at the changes I think the gif filenames are not referenced correctly in the post, e.g.
`[screenshot of map with drawn and deleted features](images/editMap_new_borders.png)` vs 
`images/edit_map_draw_new_borders.gif `

**Comment by timelyportfolio** [comment]
@tim-salabim, thanks for checking.  I just overwrote the existing images and left links/paths the same, so I believe they should works as before.  All images for posts are in https://github.com/r-spatial/r-spatial.org/tree/gh-pages/images.  Looking at a couple other posts though the path is as I would expect `/images/...` instead of `images`.

If I read this [line](https://github.com/r-spatial/r-spatial.org/blob/4acc7cc84256e6a15dd31fe649afcc9c2a22a114/_rmd/makefile#L11) correctly the `images` will be converted to `/images`.  @edzer, sorry to bother, but is this correct?  Is there a preference for `\images` or `images`?  For reference, here is a direct [link](https://github.com/timelyportfolio/r-spatial.org/blame/ea0247e883e3473766be7718b3fab1297bfef842/_rmd/2017-01-30-mapedit_intro.Rmd#L56) to one of the image path references.

**Comment by tim-salabim** [comment]
I think when find/replacing `edit_map` w `editMap` this also changed the paths, e.g. [here](https://github.com/r-spatial/r-spatial.org/pull/10/commits/ea0247e883e3473766be7718b3fab1297bfef842#diff-391f3b74040ffcc2e0459e2da5954c64R94)

**Comment by timelyportfolio** [comment]
I see now.  I thought I had changed all, but guess I didn't.

File: _rmd/2017-01-30-mapedit_intro.Rmd [hunk 1]
```diff
@@ -21,41 +21,41 @@ The R ecosystem offers a powerful set of packages for geospatial analysis.  For
 
 ## Install mapedit
 
-To run the code in the following discussion, please install with `devtools::install_github`.  Please be aware that the current functionality is strictly a proof of concept, and the API will change rapidly and dramatically.
+To run the code in the following discussion, please install with `devtools::install_github`.  Please be aware that the current functionality is strictly a proof of concept, and the API will change rapidly and dramatically.  Also [`mapview`]
```
Comment (tim-salabim) [inline _rmd/2017-01-30-mapedit_intro.Rmd]: Also [`mapview`]?

Comment (timelyportfolio) [reply _rmd/2017-01-30-mapedit_intro.Rmd]: sorry about that; thanks for proofreading

**Comment by tim-salabim** [comment]
Sorry to be nitpicking, but `breweries91` data is now in leaflet (Mentioned at the beginning of "Selecting Regions"

**Comment by tim-salabim** [comment]
Other than that, I think it is fine.

**Comment by timelyportfolio** [comment]
well darn, I completely lost a whole file from my module work :(  will need to resolve that before proceednig.

**Comment by timelyportfolio** [comment]
actually just saved in wrong place, so modules works again but not required for this post

**Comment by timelyportfolio** [comment]
found one more error and changed reference to `leaflet` for `breweries91`

**Comment by timelyportfolio** [comment]
@tim-salabim, do you agree this is ready?  @edzer, what else do we need to do to get this updated?  Thanks!

**Comment by tim-salabim** [comment]
Yes, I think this can be updated

**Comment by edzer** [comment]
Should be on-line now; pls check

**Pull request merged**